api/controllers/categoryController.js

```diff
@@ -0,0 +1,19 @@
+const categoryModel = require('../models/categoryModel'); //mongoose instance
+
+const getCategories = (req, res) => {
+    categoryModel.find({})
+        .select('title')
+        .exec((err, resp) => {
+            res.status(200).send(resp)
+        });
+};
+
+const insertCategory = (req, res) => {
+    categoryModel.create(req.body)
```

Comment:
very trusting of what's in the req.body but on the flip side it's really smooth like that